layouts/partials/nav.html

```diff
@@ -46,16 +46,23 @@
 		<a href="{{ $paginator.Next.URL }}">
 			<i class="inverted big link chevron circle right icon" title="{{ i18n "nextPage" }}"></i>
 		</a>
-		{{ else }}	
+		{{ else }}
 		<i class="disabled inverted big chevron circle right icon" title="{{ i18n "nextPage" }}"></i>
 		{{ end }}
 	</div>
-	{{ end }}	
+	{{ end }}
+        {{- if .Site.Params.enableRSS -}}
```

Comment:
This line isn't aligned properly.

Reply:
Also, put the *RSS button* code snippet after the *Sidebar button* code snippet to fix the misalignment of the RSS button on post pages.

Reply:
Yea, it is a big mistake. I will fix it.

Reply:
For the alignment, i think it if a problem come from your side :

Look at : https://github.com/UtkarshVerma/hugo-dream-plus/blob/master/layouts/partials/nav.html

You think you use tabluation and not double space. W3C recommande to use double space ( https://www.w3schools.com/html/html5_syntax.asp )
BYTW, i'will fix it.